packages/components/src/components/VirtualizedList/VirtualizedList.tsx

```diff
@@ -138,11 +140,13 @@ export function VirtualizedListComponent<T extends BaseItemProps>({
     }, [containerRef]);
 
     useEffect(() => {
-        if (isChanged(items, initialItems)) {
+        if (itemsFingerprint !== initialItemsFingerprint) {
             setItems(initialItems);
+            setItemsFingerprint(initialItemsFingerprint);
             resetScroll();
         }
-    }, [initialItems, items, resetScroll]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [initialItemsFingerprint, itemsFingerprint, resetScroll]);
```

Comment:
This caused the loosing scroll position when accounts refreshed. The `isChanged` incorrectly output true instead false. Switched to fingerprints instead (SHA3-256).